apps/total-typescript/src/lib/get-subscriber-by-email.ts

```diff
@@ -0,0 +1,10 @@
+import first from 'lodash/first'
+
+export const getSubscriberByEmail = async (email: string) => {
```

Comment:
this is a one-off function to get ck subscriber by email since that's the only connection between users db.